src/Components/Movie/Movie.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export const Movie = (props) => {
+  return (
+    <main>
+      <h1 className='title'>{props.title}</h1>
+      <img src={props.image_poster} alt={`poster for ${props.title}`}></img>
+      <h3 className='brief-description'>{props.brief_description}</h3>
+      <p className='release-date'>Released: {props.release_date}</p>
+      <p className='runtime'>Runtime: {props.runtime} minutes</p>
+      <p className='content-rating'>Content Rating: {props.content_rating}</p>
+      <p className='imdb-rating'>Audience Rating: {Math.floor(props.imdb_rating)}</p>
+    </main>
+  )
+}
+
+//do we need movie_plot and brief_description? 
+// the content seems different, but some of the movie plots are NULL, should we be 
+// prioritizing brief description only?
```

Comment:
Yeah I think thats up to us really, conditional rendering for `movie plots` ? Or just use `brief description` Or `movie plots || brief description`

It doesnt make much of a difference for me, but some of the plot descriptions can be quite long.

Reply:
Yeah, I think we'll have to see what it looks like and then we'll have more direction for our next step.